documentation/API_Release_Guidelines.md

```diff
@@ -65,68 +83,75 @@ API release numbers are GitHub tags of the format "rx.y".
-    * for the public-release, the consolidated changes since the last public-release
-  * the update of the [README.md]() (as necessary)  *
+A release PR provides only the following changes: 
+* update of the version information in the API OAS definition files (no "wip" in the version field or URL of any of the API files).
+* update the [API-Readiness-Checklist.md]() ensuring all required release assets are available. If not yet available, create a local copy under the /documentation folder and prefix it with your API Sub Project name (Copy from [ReleaseManagement/documentation/API-Readiness-Checklist.md](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md).
+* update the [CHANGELOG.md]() in the home of the API repository. If not yet available, copy from [ReleaseManagement/documentation/CHANGELOG_TEMPLATE.md](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md). See also the example available there.
+  * add a new section at the top for the release and each API version with the following content:
+    * for each first alpha or release-candidate API version, all changes since the release of the previous public API version
+    * for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
+    * for a public API version, the consolidated changes since the release of the previous public API version
+* update of the [README.md]() (as necessary)
 
-* Manage the "release PR" approval, merge the approved "release PR" and create the release
-  * An API release is created using the GitHub release feature.
-  * The release name shall be the same as the release tag and shall have the following format: rx.y
-  * The x.y number shall follow the release numbering scheme  as defined in the above section on API release numbering
-  * Outside the API Sub Project, the release name shall be the API name (for definition see the versioning section in the [API Design Guidelines](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md)) followed by the release number e.g. qod rx.y
+**Create the release**
+* manage the release PR approval
+* merge the approved release PR
+* create the release:
+  * an API release is created using the GitHub release feature (a release tag and, optionally, a release package).
+  * the release name shall be the same as the release tag and shall have the following format: rx.y
+  * the x.y number shall follow the release numbering scheme  as defined in the above section on API release numbering
+  * note: outside the API Sub Project, the release name shall be referred to with the API name (for definition see the versioning section in the [API Design Guidelines](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md)) followed by the release number e.g. quality-on-demand rx.y
```

Comment:
Not sure if we need this line here.

Reply:
You mean the line 136 ? on naming outside the project ? fine with me. (I did not put it)

Reply:
Yes, the suggestion is to delete line 136 from the document as it is a) out of scope and b) we can anyway not steer how the releases / API version will be referred "outside" 

(btw: the same sentence is in the wiki, but I haven't put it there either).

Reply:
Line 136 is now removed